packages/e2e-tests/specs/autosave.test.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	clickBlockAppender,
+	createNewPost,
+	getEditedPostContent,
+	pressKeyWithModifier,
+} from '@wordpress/e2e-test-utils';
+
+const AUTOSAVE_INTERVAL_SECONDS = 15;
```

Comment:
That's a big number of seconds to wait in the test, to be fair. Can we set something lower through some sort of override? In other words, can we avoid using sleep as much as it is possible?

Reply:
We could bring it down to 5, if that makes sense as a user experience.

It might also be beneficial to make this an editor setting so it could be customised. Would this be a requirement for this particular PR?

Reply:
How slow would setting session storage be for large posts?